lambda_functions/submission_service/anonymize_unconfimed_submissions.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import traceback
+from core_layer import helper
+from core_layer.connection_handler import get_db_session
+from core_layer.handler import submission_handler
+
+
+def anonymize_unconfirmed_submissions(event, context, is_test=False, session=None):
+
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
```

Comment:
In other files we instantiate the logger outside of the methods.
As long as this method remains the only method it is fine.
However if we add further methods to this file, this might lead to confusion.